scripts/charts.sh

```diff
@@ -57,15 +57,15 @@ run_diff() {
     ;;
   all)
     rm -rf "/tmp/charts/${chart}"
-    helm pull "${chart}" --version "${requested_version}" 2>&1 > /dev/null --untar --untardir "/tmp/charts/${chart%%/*}" 2> /dev/null
+    helm pull "${chart}" --version "${requested_version}" --untar --untardir "/tmp/charts/${chart%%/*}" 2>&1 >/dev/null 2>/dev/null
```

Comment:
I'm surprised that this even worked. :smile:

Reply:
I mean it doesn't, or well not smoothly. :sweat_smile: 

First stderr>stdout, stdout>null, stderr>null.
The stderr>stdout is unnecessary, and there's also the `&>/dev/null` that could do both in one go.